drivers/wdm/audio/legacy/stream/pnp.c

```diff
@@ -353,7 +353,15 @@ StreamClassStartDevice(
 
 
     /* First forward the request to lower attached device object */
-    Status = ForwardIrpSynchronous(DeviceObject, Irp);
+    if (IoForwardIrpSynchronously(DeviceObject->LowerDeviceObject, Irp))
```

Comment:
Should be `DeviceExtension->LowerDeviceObject` (and shouldn't compile the way it is)